src/main/cpp/CMakeLists.txt

```diff
@@ -44,9 +44,23 @@ project(
 # * build options ---------------------------------------------------------------------------------
 
 option(PER_THREAD_DEFAULT_STREAM "Build with per-thread default stream" ON)
```

Comment:
Since this is invoked only via the pom file, I don't see a need to keep backwards compatibility here.

Reply:
Thanks for the review.  I made this change.